observability-kit-starter/src/main/java/com/vaadin/observability/ObservabilityHandler.java

```diff
@@ -30,6 +41,9 @@ private static Logger getLogger() {
 
     private final String id = UUID.randomUUID().toString();
 
+    public Function<String, String> config = (key) -> null;
+    public BiConsumer<String, Map<String,Object>> exporter = (id, map) -> {};
```

Comment:
Since `ObservabilityHandler` is installed as a Session request handler, fields should either reference Serializable interfaces or be defined as transient.